### PR TITLE
Version 1

### PR DESCRIPTION
**Pull request by elliotthumphreys**

### CODE REVIEW

File: src/Models/ContactSTARTRequest.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string PhoneNumber { get; set; }
+        public string EmailAddress { get; set; }
+        public DateTime DateOfBirth { get; set; }
+        public string TimeSlot { get; set; }
+        public Address Address { get; set;  }
+    }
+}
```
Comment (ColinLees) [inline src/Models/ContactSTARTRequest.cs:36]: These should be split into three seperate files to allow easier identification of where models are

File: src/Services/ContactSTARTService.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+using System;
+using System.Threading.Tasks;
+
+namespace contact_start_service.Services
+{
+    public interface IContactSTARTService
+    {
+        Task<string> CreateCase(ContactSTARTRequest request);
+    }
+
```
Comment (ColinLees) [inline src/Services/ContactSTARTService.cs:20]: This shouhld be pulled out into it's own interface file

File: src/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddStorageProvider(Configuration);
+            services.AddResilientHttpClients<IGateway, Gateway>(Configuration);
+            services.Configure<VerintConfiguration>(settings => Configuration.GetSection("VerintConfiguration").Bind(settings));
+            services.AddSingleton<IContactSTARTService, ContactSTARTService>();
```
Comment (ColinLees) [inline src/Startup.cs:36]: Registering of services and configuration elements should be abstracted out into appropriate sevice collection extension methods

File: src/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+            services.AddAvailability();
+            services.AddSwagger();
+            services.AddHealthChecks()
+                    .AddCheck<TestHealthCheck>("TestHealthCheck");
+
+            services
+                .AddMvc()
+                .SetCompatibilityVersion(CompatibilityVersion.Version_2_2)
```
Comment (ColinLees) [inline src/Startup.cs:44]: We are using .Net Core 3.1 we shouldn't set the compatibility version to 2.2

File: src/Startup.cs [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+            app.UseMiddleware<Availability>();
+            app.UseMiddleware<ApiExceptionHandling>();
+            
+            app.UseHealthChecks("/healthcheck", HealthCheckConfig.Options);
+
+            app.UseSwagger();
+            app.UseSwaggerUI(c =>
+            {
+                c.SwaggerEndpoint("v1/swagger.json", "contact_start_service API");
```
Comment (ColinLees) [inline src/Startup.cs:75]: That name of the API at the end of this should be human readable without _

File: src/Utils/ServiceCollectionExtensions/ServiceCollectionExtensions.cs [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+namespace contact_start_service.Utils.ServiceCollectionExtensions
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static void AddSwagger(this IServiceCollection services)
+        {
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new OpenApiInfo { Title = "contact_start_service API", Version = "v1" });
```
Comment (ColinLees) [inline src/Utils/ServiceCollectionExtensions/ServiceCollectionExtensions.cs:13]: The Title needs to be human readable

File: src/Services/ContactSTARTService.cs (hunk 2 above)
Comment (elliotthumphreys) [reply src/Services/ContactSTARTService.cs]: https://github.com/smbc-digital/contact-start-service/pull/1/commits/60be4962e956ca4f7463423f8817094f231a125c

**Pull request merged**